.eslintrc.json

```diff
@@ -82,5 +82,12 @@
     "import/resolver": {
       "typescript": {}
     }
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.test.*"],
```

Comment:
Didn't know we have tests 🙃

Reply:
The `npm run test` tells me that 'no tests found'. Is that expected?

Reply:
Yes - I added the first test in the "dependent" PR: https://github.com/Cardinal-Cryptography/polkadot-staking-dashboard/pull/18